Transmission.xcodeproj/project.pbxproj

```diff
@@ -2984,9 +3021,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				CLANG_ENABLE_OBJC_ARC = NO;
-				HEADER_SEARCH_PATHS = (
-					"third-party/libevent/include",
-				);
+				HEADER_SEARCH_PATHS = "third-party/libevent/include";
```

Comment:
Xcode reformatted on save, Project view still shows that it's Xcode-8.0 compatible.